src/Block_Reservoir.f90

```diff
@@ -0,0 +1,66 @@
+module Block_Reservoir
+
+        implicit none
+        
+        ! --------------------------------------------------------------------
+        !       reservoir to RBM variables
+        !----------------------------------------------------------------------
+
+        !------------------------read in reservoir info--------------------------
+        real, dimension (:), allocatable:: dam_lat, dam_lon, res_grid_lat, res_grid_lon
+        real, dimension (:), allocatable:: res_depth_feet, res_width_feet, res_length_feet
+        integer, dimension (:), allocatable:: dam_number, start_operating_year
+        real, dimension (:), allocatable:: res_top_vol, res_bot_vol, res_max_flow, res_min_flow
+        integer, dimension (:), allocatable:: res_start_node, res_end_node
+       !  integer, dimension (:,:), allocatable:: nodes_x   !for each reach, what are all the nodes
```

Comment:
Delete comment line 15. If node_x is not used, there is no need to keep it there. Same for line 26, 57 and 58